classes/person.rb

```diff
@@ -2,14 +2,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
-  def initialize(age, name = 'Unknown', parent_permission: true)
+  def initialize(age, name = 'Unknown')
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
-    @parent_permission = parent_permission
+    @parent_permission = true
+    @rentals = []
   end
 
```

Comment:
- [ ] The `Person` class should include a method to add rentals, so the `has-many` relationship is setup correctly (a person has many rentals)